docs/conf.py

```diff
@@ -43,11 +43,21 @@
 copyright = '2019, Qiskit Development Team'  # pylint: disable=redefined-builtin
 author = 'Qiskit Development Team'
 
+import qiskit_sphinx_theme
+
 # The short X.Y version
 version = ''
 # The full version, including alpha/beta/rc tags
 release = '0.12.0'
 
+rst_prolog = """
+.. |version| replace:: {0}
+""".format(release)
+
+html_static_path = ['_static']
+templates_path = ['_templates']
+html_css_files = ['style.css']
```

Comment:
This shouldn't be needed as we can use the style sheet from the qiskit_sphinx_theme package.
```suggestion
```